.github/workflows/continuous-integration.yml

```diff
@@ -66,6 +63,52 @@ jobs:
           name: "phpunit-sqlite-${{ matrix.deps }}-${{ matrix.php-version }}.coverage"
           path: "coverage.xml"
 
+  phpunit-lower-php-versions:
+    name: "PHPUnit with SQLite"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.1"
+          - "7.2"
+        deps:
+          - "fixed"
```

Comment:
In packages where we use an update strategy, I used `"normal"` here, it makes more sense IMO.